install_config/syncing_groups_with_ldap.adoc

```diff
@@ -83,6 +83,7 @@ scope: sub <2>
 derefAliases: never <3>
 timeout: 0 <4>
 filter: (objectClass=inetOrgPerson) <5>
+pageSize: 0 <6>
```

Comment:
I would add this to all the examples on the page, so when people copy/paste into their configs, they have the param sitting there if they need to set it later